.github/workflows/run_test.sh

```diff
@@ -26,12 +26,12 @@ if [ $OS_TYPE == "c8s" ];then
     CONTAINER_IMAGE="quay.io/nmstate/c8s-nmstate-dev"
     CUSTOMIZE_ARG='--customize=
         dnf install -y python3-varlink libvarlink-util python3-jsonschema \
-            python3-nispor python3-openvswitch2.11;'
+            python3-nispor python3-openvswitch2.11 teamd NetworkManager-team;'
 elif [ $OS_TYPE == "ovs2_11" ];then
     CONTAINER_IMAGE="quay.io/nmstate/c8s-nmstate-dev"
     CUSTOMIZE_ARG='--customize=
         dnf install -y python3-varlink libvarlink-util python3-jsonschema \
-            python3-nispor;
+            python3-nispor teamd NetworkManager-team;
```

Comment:
Restart NetworkManager ?

Reply:
The `start_machine_services()` will restart the daemon